crates/matrix-sdk-crypto/src/olm/group_sessions/forwarder_data.rs

```diff
@@ -0,0 +1,159 @@
+// Copyright 2025 The Matrix.org Foundation C.I.C.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use ruma::{DeviceId, UserId};
+use serde::{Deserialize, Serialize};
+
+use crate::olm::{KnownSenderData, SenderData};
+
+/// Represents information about the user who forwarded a megolm session under
+/// [MSC4268]. This is similar to [SenderData], but it is limited to variants
```

Comment:
I think our normal style is to use a fixed-width font for identifier names:

```suggestion
/// [MSC4268]. This is similar to [`SenderData`], but it is limited to variants
```

(likewise below)